app/src/components/fields/AutocompleteField.tsx

```diff
@@ -30,7 +30,7 @@ const AutocompleteField: React.FC<IAutocompleteField<string | number>> = <T exte
     const result = props.options.find((option) => existingValue === option.value);
 
     if (!result) {
-      return null as unknown as IAutocompleteFieldOption<T>;
+      return (null as unknown) as IAutocompleteFieldOption<T>;
```

Comment:
(not this line specifically) It looks like some formatting is changing here and in some other files in the app. I believe the most recent version of the prettier formatter actually removes these brackets, so it might be that you need to run `make install` to ensure you have the latest version and then re-run the formatter, as I think there should be no formatting related changes here.